private-templates-service/client/src/components/Dashboard/PreviewModal/TemplateInfo/VersionCard/styled.tsx

```diff
@@ -38,11 +38,16 @@ export const VersionCardRow = styled.div`
   display: flex;
   width: 100%;
   justify-content: space-around;
-  margin-top: 8px;
-  margin-left: 16px;
-  margin-bottom: 8px;
-  margin-right: 16px;
+  padding-top: 8px;
+  padding-left: 16px;
+  padding-bottom: 8px;
+  padding-right: 16px;
```

Comment:
can you format this padding to:  `padding: 8px 16px 8px 16px; ` ( top right bottom left)  ?